src/core/server/deprecations/routes/get.ts

```diff
@@ -5,8 +5,7 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
```

Comment:
NIT: add a blank line between the _License header_ and the first import.

Reply:
We have a linter to fix that kind of thing, if it is not in the linter then it is OK since we don't have a kibana wide preference for this